src/components/Shared/MessageHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import getAvatar from '@lib/getAvatar';
+import type { FC } from 'react';
+import { useState } from 'react';
+import { useAppStore } from 'src/store/app';
+
+import Follow from './Follow';
+
+const MessageHeader: FC = () => {
```

Comment:
I wonder if this would be better as a stateless component that takes data in as props. Similar to the `MessagesList`. Would be nice to handle all the state management in the parent.

Reply:
Resolved